init.gradle

```diff
@@ -0,0 +1,23 @@
+allprojects {
+    ext {
+        // Properties for standalone build.
+        // These will be overridden by root project values to avoid interfering with standard build.
+        artifactory_contextUrl = "https://artifactory.labkey.com/artifactory"
+
+        // The source and target versions of Java for compilation tasks
+        // @JavaRuntimeVersion
+        sourceCompatibility = 14
+        targetCompatibility = 14
+
+        labkeyVersion = "21.3-SNAPSHOT"
+        servletApiVersion = "3.0"
+        labkeyClientApiVersion = "1.3.2"
+
+        apacheTomcatVersion = "8.5.51"
```

Comment:
Any reason not to put these version properties into the `gradle.properties` file?  That will be the more obvious place to look for them.